src/application/engine/Menu.java

```diff
@@ -1,4 +1,74 @@
 package application.engine;
 
+import application.coffe.Coffee;
+
+import java.util.Scanner;
+
 public class Menu {
+
+    public void run() {
+        Scanner scanner = new Scanner(System.in);
+        Machine machine = new Machine();
+        String result = "";
+        boolean control = true;
+        while (control) {
+            System.out.println("Write action (buy, fill, take, remaining, exit):");
+            var response = scanner.next();
+            switch (response) {
+                case "buy":
+                    result = makingCoffee(scanner, machine, result);
+                    break;
+                case "fill":
+                    makeFill(scanner, machine);
+                    result ="";
+                    break;
+                case "take":
+                    result = machine.take();
+                    break;
+                case "remaining":
+                    result = machine.status();
+                    break;
+                default:
+                    result = "No such command";
+                case "exit":
+                    control = false;
+            }
+            System.out.println(result);
```

Comment:
It is better to call System.out.println in cases where it is needed.
Currently for some cases an empty line will be printed